appsembler/edx-analytics-dashboard
# (WIP) Appsembler theme for Analytics

# Basic Branding
<img width="350" src="https://user-images.githubusercontent.com/645156/31227128-6486da16-a9e1-11e7-838f-7cd2a56ec691.png" />


# TODO
 - [ ] Get it reviewed by Matej
 - [ ] Resize the logos to fit (after Matej reviews/updates it)
 - [ ] Get it reviewed by another engineer
 - [x] Finish the configuration repo PR for the rest of the branding (https://github.com/noderabbit-team/edx-configs/pull/167)

# How to Test

 - Clone this repo `git@github.com:appsembler/edx-analytics-dashboard.git`
 - Follow the installation guide in the [Ficus README.md](https://github.com/appsembler/edx-analytics-dashboard/tree/open-release/ficus.master). The steps are mainly:
   - `$ mkvirtualenv`
   - `$ make develop`
   - `$ make migrate`
 - Run the following command: `$ THEME_CSS=sass/themes/appsembler.scss ./manage.py runserver 0.0.0.0:9000`

**Hint:** Try `edx.scss`, `open-edx.scss` and `appsembler.scss` to see the differences.

## What to Test
 - Open the homeapge: `http://localhost:9000/` (notice the new logo and the aligned login button)
 - Open the authentication error page `http://localhost:9000/403/` (notice the negative logo)

## What's not to Test
 - The email, since it's configurable using Ansible, so a PR should be created in the https://github.com/noderabbit-team/edx-configs/ repo.
 - The company name (Open edX vs. Appsembler), also configurable using Ansible.
